### PR TITLE
Consistent styling for in-content links

### PR DESCRIPTION
**Pull request by MattIPv4**

### CODE REVIEW

**Review by pjeweb — COMMENTED** [review]
If you could figure how to prevent wrapping right before the icon (maybe using a non-breaking space `\u00A0`/`&nbsp;`).

ambassador iucn link wrapping just before icon

**Comment by MattIPv4** [comment]
Hm, I'm not sure it's possible to stop it wrapping? Even if you `display: inline` it, it will still wrap?

File: apps/website/src/components/content/Link.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  return (
+    <NextLink {...props}>
+      {children}
+      {external && !custom && (
+        <IconExternal
+          size="0.75em"
+          className="ml-1 mr-0.5 inline-block align-baseline"
+        />
+      )}
```
Comment (pjeweb) [inline apps/website/src/components/content/Link.tsx:41]: Try this:
```typescript
      {external && !custom && (
        <>
          {"\u00A0"}
          <IconExternal
            size="0.75em"
            className="ml-1 mr-0.5 inline-block align-baseline"
          />
        </>
      )}
```

Comment (MattIPv4) [reply apps/website/src/components/content/Link.tsx:41]: Yah, no dice with that, it still wraps

Comment (pjeweb) [reply apps/website/src/components/content/Link.tsx:41]: wait, formattig broke it again

Comment (pjeweb) [reply apps/website/src/components/content/Link.tsx:41]: no that works locally for me

Comment (pjeweb) [reply apps/website/src/components/content/Link.tsx:41]: i'm dumb i resized the browser

**Review by pjeweb — APPROVED** [review]

**Pull request merged**